### PR TITLE
devonfw/devon4ng-ionic#DEVC-211

### PR DESCRIPTION
**Pull request by jambulud**
devonfw/devon4ng-ionic-application-template#DEVC-211
Migration from ionic V3 to V4

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,31 +1,22 @@
...
+      "dom"
     ]
-  },
-  "include": [
-    "src/**/*.ts"
-  ],
-  "exclude": [
-    "node_modules",
-    "src/**/*.spec.ts",
-    "src/**/__tests__/*.ts"
```
Comment (jdiazgon) [inline tsconfig.json]: I would not remove the include and exclude from the tsconfig

**Comment by sjimenez77** [comment]
Please execute a `ng lint` and fix the errors. If you use Prettier as we recommend the issues will be solved very quickly.

File: .gitignore [hunk 2]
```diff
@@ -13,21 +13,20 @@ log.txt
 npm-debug.log*
 
 .idea/
+.ionic/
 .sourcemaps/
 .sass-cache/
 .tmp/
 .versions/
 coverage/
-dist/
```
[the remark this answers is not in the document]
Comment (jambulud) [reply .gitignore]: added

File: config.xml [hunk 3]
```diff
@@ -1,89 +0,0 @@
-<?xml version='1.0' encoding='utf-8'?>
-<widget id="io.ionic.starter" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
-    <name>oasp-ionic-application-template</name>
```
[the remark this answers is not in the document]
Comment (jambulud) [reply config.xml]: Deleted because that file is used by Cordova but not Capacitor

File: tsconfig.json (hunk 1 above)
Comment (jambulud) [reply tsconfig.json]: Added again

File: .editorconfig [hunk 4]
```diff
@@ -1,17 +0,0 @@
-# EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs
```
[the remark this answers is not in the document]
Comment (jambulud) [reply .editorconfig]: File added again

**Comment by sjimenez77** [comment]
Tested successfully.

**Pull request merged**